static/dream_web/index.css

```diff
@@ -1,61 +1,268 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap';

```

Comment:
Why is this `import` here if the "Roboto" font isn't even used in the CSS?

Your `body {}` has a `font-family: 'Montserrat', sans-serif;`. I'd change the `body` CSS to have `font-family: 'Roboto', Arial, Helvetica, sans-serif; ` if you want to use Roboto.

Also remove the import from `fonts.googleapis.com`. We don't need to send a request to Google servers when we load the local UI  just because we're missing a non-essential font. It also will fail if the users are using SD offline.

Reply:
I agree with this comment. I'm personally using a GPU that is in an HPC cluster and can't access the internet, and have been working hard to work around all the other places where stable diffusion tries to download model or parameter files.

Reply:
@SergioDiazF Just checking in. Are you planning to address the change requests? I'll keep the PR open until you confirm one way or another.

Reply:
I confirm the change, I fully agree with the needs.